pkgs/development/ocaml-modules/cbor/default.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildDunePackage,
+  fetchurl,
+  ocplib-endian,
+  yojson,
+}:
+
+buildDunePackage rec {
+  pname = "cbor";
+  version = "0.5";
+
```

Comment:
You may want to set `minimalOCamlVersion` as this library won’t build with OCaml < 4.07.